src/main/java/pl/targosz/homework/Shape/Circle.java

```diff
@@ -0,0 +1,13 @@
+package pl.targosz.homework.Shape;
+
+public class Circle extends Shape{
+    double r;
```

Comment:
To pole powinno być private (analogicznie w przypadku innych klas w tym commicie). Ustawić powinieneś je za pomocą konstruktora. ;)

https://1024kb.pl/nauka-java/hermetyzacja-java/